server/routes/index.ts

```diff
@@ -266,7 +266,7 @@ export default function routes(services: Services): Router {
   router.use(
     `${basePath}/money`,
     getPrisonerData(services, { minimal: true }),
-    permissionsGuard(services.permissionsService.getMoneyPermissions),
+    prisonerPermissionsGuard(prisonPermissionsService, { requestDependentOn: [PrisonerMoneyPermission.read] }),
```

Comment:
Just this one permission to switch over, everything else is deleting redundant code